waspc/cli/src/Wasp/Cli/Command/Start/Db.hs

```diff
@@ -132,6 +133,8 @@ startPostgreDevDb waspProjectDir appName = do
           Dev.Postgres.defaultDevPass
           Dev.Postgres.defaultDevUser
           dbName
+
+  throwIfDockerDaemonIsNotRunning
```

Comment:
We will want to put this check together with other checks at the start of this function, so I would put it right after

```
throwIfExeIsNotAvailable
    "docker"
    "To run PostgreSQL dev database, Wasp needs `docker` installed and in PATH."
```

That way we have to docker checks first, then a check for occupied port, and then if all of them pass, we go on and run the database!